tests/conftest.py

```diff
@@ -6664,6 +6664,88 @@ def teardown():
     return factory
 
 
+@pytest.fixture(scope="class")
+def dr_workloads_on_managed_clusters(request):
+    """
+    Deploying subscription apps on both primary and secondary managed clusters
```

Comment:
It's not clear from the function name that it's specific to subscription based apps only.

Reply:
I think the doc string says it all, In future this func can be enhanced for appset apps also, hence the func name is generic just like dr_worklods(), If you still need let me know

Reply:
ack